src/NServiceBus.Core/Recoverability/Recoverability.cs

```diff
@@ -0,0 +1,170 @@
+        {
+            var errorQueue = context.Settings.ErrorQueueAddress();
+            context.Settings.Get<QueueBindings>().BindSending(errorQueue);
+
+            var transportTransactionMode = context.Settings.GetRequiredTransactionModeForReceives();
+            var failureInfoStorage = new FailureInfoStorage(context.Settings.Get<int>(FailureInfoStorageCacheSizeKey));
+            var localAddress = context.Settings.LocalAddress();
+
+            context.Pipeline.Register("MoveFaultsToErrorQueue", b => new MoveFaultsToErrorQueueBehavior(
+                b.Build<CriticalError>(),
+                localAddress,
+                transportTransactionMode,
+                failureInfoStorage), "Moves failing messages to the configured error queue");  //context.Pipeline.Register(new MoveFaultsToErrorQueueBehavior.Registration(context.Settings.LocalAddress(), transportTransactionMode, failureInfoStorage));
+
+            context.Pipeline.Register("AddExceptionHeaders", new AddExceptionHeadersBehavior(), "Adds the exception headers to the message");
+            context.Pipeline.Register(new FaultToDispatchConnector(errorQueue), "Connector to dispatch faulted messages");
+
+            if (IsDelayedRetriesEnabled(context.Settings))
+            {
+                var retryPolicy = GetDelayedRetryPolicy(context.Settings);
+
+                context.Pipeline.Register("SecondLevelRetries", b => new SecondLevelRetriesBehavior(retryPolicy, localAddress, failureInfoStorage), "Performs second level retries");
+            }
+
+            if (IsImmediateRetriesEnabled(context.Settings))
+            {
+                var maxRetries = GetMaxRetries(context.Settings);
+                var retryPolicy = new FirstLevelRetryPolicy(maxRetries);
+
+                context.Pipeline.Register("FirstLevelRetries", b => new FirstLevelRetriesBehavior(failureInfoStorage, retryPolicy), "Performs first level retries");
+            }
+
+            RaiseLegacyNotifications(context);
+        }
+
+        static SecondLevelRetryPolicy GetDelayedRetryPolicy(ReadOnlySettings settings)
+        {
+            Func<SecondLevelRetryContext, TimeSpan> customRetryPolicy;
+            if (settings.TryGet(SlrCustomPolicy, out customRetryPolicy))
+            {
+                return new CustomSecondLevelRetryPolicy(customRetryPolicy);
+            }
+
+            var numberOfRetries = settings.Get<int>(SlrNumberOfRetries);
+            var timeIncrease = settings.Get<TimeSpan>(SlrTimeIncrease);
+
+            var retriesConfig = settings.GetConfigSection<SecondLevelRetriesConfig>();
+            if (retriesConfig != null)
+            {
+                numberOfRetries = retriesConfig.Enabled ? retriesConfig.NumberOfRetries : 0;
+                timeIncrease = retriesConfig.TimeIncrease;
+            }
+
+            return new DefaultSecondLevelRetryPolicy(numberOfRetries, timeIncrease);
+        }
+
+        bool IsDelayedRetriesEnabled(ReadOnlySettings settings)
```

Comment:
Found it, the old code used to have dependson 

https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/Recoverability/SecondLevelRetries/SecondLevelRetries.cs#L17

I changed that to optionally but missed to check if it's enabled or not. Will fix!